tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -389,18 +389,18 @@ error: Cannot access static property \\"m\\" from instance
    |     ^ Cannot access static property \\"m\\" from instance
 
 
-error: Unknown symbol \\"this\\"
-  --> ../../../examples/tests/invalid/access_static_from_instance.test.w:7:5
+error: Unknown symbol \\"instanceField\\"
+  --> ../../../examples/tests/invalid/access_static_from_instance.test.w:7:10
```

Comment:
So technically this is still just as invalid as it was before, however the error messages are different since now "this" is not an unknown symbol.

@Chriscbr @eladb just curious about your thoughts, I kinda wonder if it would be less confusing to introduce something like "app" (mentioned in the original PR).

Reply:
I don't have a strong intuition for which will be better DX, so I'm open to either

Reply:
Yea, I dunno thinking about it more my gut feeling is that "this" would be the better DX over something like "app" but yea maybe we just try `this` (pun intended) and change as needed